src/components/Facility/FacilityHome.tsx

```
@@ -105,7 +107,17 @@ export const FacilityHome = ({ facilityId }: Props) => {
       queryClient.invalidateQueries({
         queryKey: ["facility", facilityId],
       });
-      navigate("/");
+
+      if (history.length > 1) {
+        const prevPath = history[1];
+        const orgMatch = prevPath.match(/\/org\/([^/]+)\/facilities/);
+
+        if (orgMatch && orgMatch[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)